tv-shows-ratting-app/src/app/layout.tsx

```diff
@@ -0,0 +1,32 @@
+import type { Metadata } from "next";
+import { Inter } from "next/font/google";
+import { Providers } from "./providers";
+import { Header } from "@/components/shared/Header/Header";
+import { Box } from "@chakra-ui/react";
+
+const inter = Inter({ subsets: ["latin"] });
+
+export const metadata: Metadata = {
+  title: "Create Next App",
+  description: "Generated by create next app",
+};
+
+//kako bi se postavila odgovarajuca boja na neki token?
+export default function RootLayout({
+  children,
+}: Readonly<{
+  children: React.ReactNode;
+}>) {
+  return (
+    <html lang="en">
+      <body className={inter.className}>
+        <Providers>
+          <Box bg="#200050" color='white' paddingLeft={4} paddingRight={4} paddingBottom={4}> 
```

Comment:
Ovo je okej ali mogli smo ovo i kraće napisati:
```suggestion
          <Box bg="#200050" color='white' px={4} paddingBottom={4}> 
```